packages/server/src/instances/service.ts

```diff
@@ -96,18 +114,20 @@ export class InstanceService extends Service {
     try {
       await instance.initialize()
     } catch (e) {
-      instance.logger.error('Error trying to initialize conduit.', (e as Error).message)
       this.cache.del(conduitId)
 
       // TODO: replace by HealthService
```

Comment:
What do you mean by 'replace by HealthService'. Isn't it what you indirectly do on line 126?

Reply:
Good point. I updated the comment to say "Replace by StatusService". I want to replace the tracker of failures. I initially thought this would be the health service but it'll be the StatusService instead.